README.md

```diff
@@ -46,50 +61,36 @@ layouts (see the [Layout Plugin](https://www.drupal.org/project/layout_plugin)
 module) or install a contributed library of layouts like
 [Radix Layouts](https://www.drupal.org/project/radix_layouts).
 
-## Workflow
+### Workflow
 Lightning includes tools for building organization-specific content workflows.
 Out of the box, Lightning gives you the ability to manage content in one of four
 workflow states (draft, needs review, published, and archived). You can create
 as many additional states as you like and define transitions between them. It's
 also possible to schedule content (either a single node or many at once) to be
 transitioned between states at a specific future date and time.
 
-## Preview (Experimental)
+### Preview (Experimental)
 The Workspace Preview System (WPS) gives site builders, editors, authors, and
 reviews the ability to send collections of content through an editorial workflow
 and preview that content within the context of the current live site. WPS is a
 collection of contributed Drupal modules with additional configuration UX
 improvements that all just works out of the box.
 
-## Installing Lightning
-The preferred way to install Lightning is using our
-[Composer-based project template][template]. It's easy!
-
-If you don't want to use Composer, you can install Lightning the traditional way
-by downloading a tarball from our
-[drupal.org project page](https://www.drupal.org/project/lightning).
-
-You can customize your installation using [lightning.extend.yml](https://github.com/acquia/lightning/blob/8.x-1.x/lightning.extend.yml). Using this file, you can choose which Lightning features to install as well as install your own custom features.
-
 ## Project Roadmap
-The roadmap is subject to change, but our projected schedule is:
-
-* QTR3/4, 2016
-  * Point in time preview
-  * Personalization
-  * Remote replication (store workspaces on external apps)
-  * Search API integration
-
-You can also look for general enhancements along the way. Please use the
-[Lightning issue queue][issue_queue] for latest information and to request
-features or bug fixes.
+We publish sprint plans for each patch release. You can find a link to the
+current in our [Meta Lightning Releases][meta_releases]
```

Comment:
Should be "you can find a link to the current one in this meta-issue on drupal.org"